src/ztc/api/serializers/besluittype.py

```diff
@@ -1,21 +1,24 @@
+from django.utils.translation import ugettext_lazy as _
+
 from rest_framework import serializers
+from rest_framework.exceptions import PermissionDenied
 
-from ...datamodel.models import BesluitType
+from ...datamodel.models import BesluitType, InformatieObjectType, ZaakType
 
 
 class BesluitTypeSerializer(serializers.HyperlinkedModelSerializer):
     informatieobjecttypes = serializers.HyperlinkedRelatedField(
         view_name='informatieobjecttype-detail',
         many=True,
         lookup_field='uuid',
-        read_only=True
+        queryset=InformatieObjectType.objects.all()
     )
 
     zaaktypes = serializers.HyperlinkedRelatedField(
         many=True,
-        read_only=True,
         view_name='zaaktype-detail',
         lookup_field='uuid',
+        queryset=ZaakType.objects.all()
```

Comment:
Same here.

Reply:
implicit `source` here matches the name of the field i.e source = `instance.zaaktypes`
Unittest to cover this case is added